src/main/java/nl/tudelft/ewi/devhub/server/database/controllers/Deliveries.java

```diff
@@ -74,15 +78,10 @@ public boolean lastDeliveryIsApprovedOrDisapproved(Assignment assignment, Group
                 .list(QDelivery.delivery);
     }
 
-    /**
-     * Get the most recent delivery for every group in this assignment
-     * @param assignment current assignment
-     * @return a list of deliveries
-     */
     @Transactional
-    public List<Delivery> getLastDeliveries(Assignment assignment) {
+    private List<Delivery> getLastDeliveriesList(Assignment assignment, Predicate expression) {
```

Comment:
Please don't break existing API's, they could be used in PR's or Freemarker.  If we think they are unused, we should annotate them and log a warning if used before removing them.